docs/user-guide/zonal-average.ipynb

```diff
@@ -7,7 +7,7 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    ax1.set_title("Global Field")
</pre><p>This global plot is not showing up in the figure currently.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/1345/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>